pacta/build_with_tag.sh

```diff
@@ -38,17 +38,27 @@ then
     exit 2
 fi
 
+# Clone
 for repo in $repos
 do
-    # Clone
     remote="${url}${repo}.git"
```

Comment:
oh, also here

Reply:
I'm undecided. With my little experience `"$this $looks $safe"` but `"$this$looks$weird"` so I prefer `"${this}${syntax}"`.
Any suggestion?

Reply:
yeah, that makes sense to me.... if there's no whitespace around it, maybe safer to use `{}`

can totally see how `$foobar` is not the same as `${foo}bar`